src/blocker.rs

```diff
@@ -155,54 +157,69 @@ impl Blocker {
                 self.filters.check(request, &request_tokens)
             });
 
-        let exception = filter.as_ref().and_then(|f| {
-            // Set `bug` of request
-            if !f.is_important() {
-                #[cfg(feature = "metrics")]
-                print!("exceptions\t");
-                if f.has_bug() {
+        let exception = if skip_exception {
+            None
+        } else {
+            match filter.as_ref() {
+                None => self.exceptions.check(request, &request_tokens),
```

Comment:
so in the default mode of operation, `skip_unimportant = false` and `skip_exception = false` this will still check all exceptions when no filter matches?

Reply:
Yes, because exceptions are checked before other filters anyways.

Reply:
Would it make sense to skip exceptions after an important rule has been encountered, within the same engine? And likewise to skip normal rules if any important or exception rules were encountered? This would change some `BlockerResult` outputs.